src/main/java/edu/uci/ics/crawler4j/fetcher/PageFetcher.java

```diff
@@ -289,9 +289,6 @@ public PageFetchResult fetchPage(WebURL webUrl)
                     long size = fetchResult.getEntity().getContentLength();
                     if (size == -1) {
                         Header length = response.getLastHeader("Content-Length");
-                        if (length == null) {
-                            length = response.getLastHeader("Content-length");
-                        }
```

Comment:
Wow

Didn't see that.

Well, at least some sort of in-code comment should be put there as this is a very confusing string difference